### ISSUE TITLE
Upgrade room banner showing for a v3 room

### ISSUE THREAD

**Issue by Half-Shot** [vector-im/riot-web#8876]
I even checked that the state includes a m.room.create with a version of 3. This is on Riot 1.0.1

Synapse version: 0.99

**Comment by turt2live** [comment]
Please rageshake from a fresh start of your client and after seeing the warning. 

Also synapse version please.

**Comment by Half-Shot** [comment]
My shake is at https://github.com/matrix-org/riot-web-rageshakes/issues/1237

**Comment by turt2live** [comment]
@Half-Shot I'm afraid I haven't seen this since landing the capabilities stuff. Is your synapse up to date / are you still seeing this? Unfortunately the rageshake doesn't reveal anything.

**Comment by Half-Shot** [comment]
I'm slightly intimidated by the number of labels there

**Comment by turt2live** [comment]
I'm presuming this fixed - if it happens again, please reopen or open a new issue.

**Comment by Half-Shot** [comment]
Happened again 

![image](https://user-images.githubusercontent.com/2072976/56567689-60d79700-65ad-11e9-90f0-275c738418c5.png)

**Comment by Half-Shot** [comment]
I should have posted this on #9225, but it looks like it got fixed in a js-sdk update so will wait on that before squawking more.
